modules/persistence/src/services/entries/index.ts

```diff
@@ -7,14 +8,19 @@ const COLLECTION_NAME = 'documents';
 // Service responsible for memoization of page level documents.
 // Any extraneous logic performed on page level documents as part of upload should be added here
 // or within subfolders of this module
-const entriesFromZip = async (zip) => {
+const entriesFromZip = async (zip: AdmZip) => {
   const zipEntries = zip.getEntries();
   return zipEntries
     .filter((entry) => entry.entryName?.startsWith('documents/'))
     .map((entry) => deserialize(entry.getData()));
```

Comment:
minor. these are not getting the created_at fields upon upload:
![image](https://user-images.githubusercontent.com/13054820/193848042-86289907-9e51-46a9-9121-b697cb81b16b.png)

Reply:
Great catch, added as a prop on the object, derived from the build_id